revolt/http.py

```diff
@@ -105,6 +106,78 @@ async def send_message(self, channel: str, content: Optional[str], embeds: Optio
 
         return await self.request("POST", f"/channels/{channel}/messages", json=json)
 
+    def edit_message(self, channel: str, message: str, content: str) -> Request[None]:
+        json = {"content": content}
+        return self.request("PATCH", f"/channels/{channel}/messages/{message}", json=json)
+
+    def delete_message(self, channel: str, message: str) -> Request[None]:
+        return self.request("DELETE", f"/channels/{channel}/messages/{message}")
+
+    def fetch_message(self, channel: str, message: str) -> Request[MessagePayload]:
+        return self.request("GET", f"/channels/{channel}/messages/{message}")
+    
+    def fetch_messages(
+        self, 
+        channel: str, 
+        sort: SortType,
+        *, 
+        limit: Optional[int] = None, 
+        before: Optional[str] = None, 
+        after: Optional[str] = None, 
+        nearby: Optional[str] = None, 
+        include_users: Optional[bool] = None
+    ) -> Request[Union[list[MessagePayload], MessageWithUserData]]:
```

Comment:
It would be better to use an overload here so we can have finer control over the type